src/wolfgame.py

```diff
@@ -7643,12 +7674,6 @@ def start(cli, nick, chan, forced = False, restart = ""):
             if var.STASISED_ACCS[acc] <= 0:
                 del var.STASISED_ACCS[acc]
 
-    if not botconfig.DEBUG_MODE or not var.DISABLE_DEBUG_MODE_REAPER:
-        # DEATH TO IDLERS!
-        reapertimer = threading.Thread(None, reaper, args=(cli,var.GAME_ID))
-        reapertimer.daemon = True
-        reapertimer.start()
-
```

Comment:
The reaper thread only lasts for a single game (e.g. while var.GAME_ID remains unchanged). var.GAME_ID changes both when a game is first created via !join as well as when a game is !started. So removing this means you just disabled the in-game reaper, nice job breaking it hero.